remoteit/desktop
# [DESK-1125] transfer device

### Description

[DESK-1125] transfer device


### Test plan



1. Sign in
2. Go to devices
3. Select a device
4. Click in the rigth 3 dots
5. Click on Transfer Device
6. You should see the new workflow to transfer device

### Ticket

https://remoteit.atlassian.net/browse/DESK-1125


[DESK-1125]: https://remoteit.atlassian.net/browse/DESK-1125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
